code/modules/mob/living/carbon/alien/humanoid/emote.dm

```diff
@@ -104,8 +119,14 @@
 			m_type = 1
 			message = "<B>\The [src]</B> does a flip!"
 			src.SpinAnimation(5,1)
+		if("superflip")
+			m_type = 1
+			message = "<B>The [src]</B> starts flipping!"
+			src.reagents.add_reagent("fliptonium", 100)
```

Comment:
Considering it's a joke emote, It's kind of expected to be bad, However, If I add it to the cd list and make it add 1u instead of 100, then the same affect is accomplished without the risk for power gaming as the chem will be processed prior to being able to add more and the flipping animation will be started as expected